locale/en/docs/index.md

```diff
@@ -42,8 +42,9 @@ This documentation describes the built-in modules provided by Node.js. It does n
 
 ## ES6 features
 
-The [ES6 section](/en/docs/es6/) describes the three ES6 feature groups, and details which features are enabled by default in Node.js, alongside explanatory links. It also shows how to find which version of V8 shipped with a particular Node.js release.
+The [ES6 section](/en/docs/es6/) describes the three ES6 feature groups, and details which features are enabled by default in Node.js, alongside explanatory links. It also shows how to determine the version of  V8 which shipped with a particular Node.js release.
```

Comment:
@42,# isn't showing up highlighted by GitHub's mobile app as being part of the edits which I made; only @45, however I do take personal exception to conflating Node.js with a ```,``` because such visual syntax changes the actual local schema; it had been allowed in the OP, so I'm not about to retool an entire paragraph's wording simply to correct grandfathered-asthetic of non-hyperlinked terminology.

Reply:
GH wasn’t and still isn’t being convinced into revealing the offending d.s. but in multiple instances has decided to obscure its existence to me by incorporating it into an asymmetric line break in order to maintain the (false) appearance of
full-left-justification.